cg/server/admin.py

```diff
@@ -323,7 +324,17 @@ class PoolView(BaseView):
 class SampleView(BaseView):
     """Admin view for Model.Sample"""
 
-    column_exclude_list = ["invoiced_at"]
+    column_exclude_list = [
+        "age_at_sampling",
+        "from_sample",
+        "invoiced_at",
+        "loqusdb_id",
+        "organism",
```

Comment:
Let's keep this one. It is good for the microbial wgs samples

Reply:
Ok, I we can keep the reference-genome column visible as well I guess they're used together

Reply:
I double checked with TN and she said that we should keep both of them